tests/integration/records/test_records_serializers_latex.py

```diff
@@ -0,0 +1,567 @@
+# Copyright (C) 2014-2018 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+from factories.db.invenio_records import TestRecordMetadata
+
+
+def test_latex_serializer_serialize_EU(isolated_api_client):
+    record_json = {
+        '$schema': 'http://localhost:5000/schemas/records/hep.json',
+        'control_number': '64209',
+        'texkeys': [
+            'a123bx'
+        ],
+        'titles': [
+            {
+                'title': 'The Amazing Spiderman',
+            },
+        ],
+        'authors': [
+            {
+                'full_name': 'Lee, Stan',
+            },
+            {
+                'full_name': 'Ditko, Steve',
+            },
+            {
+                'full_name': 'Lieber, Larry'
+            }
+        ],
+        'collaborations': [
+            {
+                'value': 'LHCb',
+            },
+            {
+                'value': 'Marvel',
+            },
+            {
+                'value': 'Comics',
+            }
+        ],
+        'dois': [
+            {
+                'value': '10.1088/1361-6633/aa5514',
+            },
+        ],
+        'arxiv_eprints': [
+            {
+                'value': '1607.06746',
+                'categories': ['hep-th']
+            },
+        ],
+        'publication_info': [{
+            'journal_title': 'Phys.Rev.A.',
+            'journal_volume': '58',
+            'journal_issue': '120',
+            'page_start': '500',
+            'page_end': '593',
+            'artid': '17920',
+            'year': '2014'
+        }],
+        'report_numbers': [{
+            'value': 'DESY-17-036'
+        }]
+    }
+    TestRecordMetadata.create_from_kwargs(
+        json=record_json, index_name='records-hep')
+    response = isolated_api_client.get(
+        '/literature/64209',
+        headers={'Accept': 'application/x-latex-eu'}
+    )
+
+    expected = "%\\cite{a123bx}\n\\bibitem{a123bx}\nS.~Lee \\textit{et al.} [LHCb, Marvel and Comics]\n%``The Amazing Spiderman,''\nPhys.\\ Rev.\\ A.\\  \\textbf{58} (2014) no.120, 500-593\ndoi:10.1088/1361-6633/aa5514\n[arxiv:1607.06746 [hep-th]]."
```

Comment:
can you make this a multiline string (in all tests)? [`textwrap.dedent`](https://docs.python.org/2/library/textwrap.html#textwrap.dedent) might be useful.